src/mlpack/methods/hoeffding_trees/hoeffding_tree_main.cpp

```diff
@@ -159,6 +159,10 @@ static void mlpackMain()
       model = new HoeffdingTreeModel(HoeffdingTreeModel::INFO_HOEFFDING);
     else if (CLI::HasParam("info_gain") && (numericSplitStrategy == "binary"))
       model = new HoeffdingTreeModel(HoeffdingTreeModel::INFO_BINARY);
+    else {
```

Comment:
Avoid placing opening braces on the same line. See https://github.com/mlpack/mlpack/wiki/DesignGuidelines#brace-placement for more information; this might be irrelevant if we switch the ``else if`` to ``else``.

Reply:
Thanks for clarifying, I just updated the pull request with a new commit.